tests/vmi_configuration_test.go

```diff
@@ -1376,7 +1376,7 @@ var _ = Describe("[sig-compute]Configurations", func() {
 				}, 240*time.Second, 2).Should(BeTrue(), "Should have guest OS Info in vmi status")
 
 				Expect(err).ToNot(HaveOccurred())
-				Expect(updatedVmi.Status.GuestOSInfo.Name).To(Equal("Fedora"))
+				Expect(updatedVmi.Status.GuestOSInfo.Name).To(ContainSubstring("Fedora"))
```

Comment:
We are supposed to know the expect name, otherwise bugs may get in without noticing. Can this be made more explicit?

Reply:
If fedora 35 its `Fedora Linux`. I thought it would be enough to search only for the `Fedora` substring. That will make the test less 'fragile' and alow easier transition to new images.

Reply:
I understand, but on the other side it may masks bugs because it is too loose.
It is worth another opinion on this from the ones who maintain these tests.